app/views/admin/cards/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1> Create a NPC card </h1>
+
+<%= form_for @card, url: admin_cards_path do |f| %>
```

Comment:
Let's check out a gem called simple_form.   It will make these type of forms WAY clearer.